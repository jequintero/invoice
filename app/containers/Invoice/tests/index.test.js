import React from 'react';
import { shallow } from 'enzyme';
// import { enzymeFind } from 'styled-components/test-utils';

import { Invoice } from '../index';

describe('<Invoice />', () => {
  it('Expect to have unit tests specified', () => {
    const wrapper = shallow(<Invoice tableHeaders={[]} tableData={[]} />);
    expect(wrapper).toMatchSnapshot();
  });
});
