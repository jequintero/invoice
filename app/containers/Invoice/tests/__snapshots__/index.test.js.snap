// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Invoice /> Expect to have unit tests specified 1`] = `
<div>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Invoice Editor
    </title>
    <meta
      content="Invoice Editor"
      name="description"
    />
  </HelmetWrapper>
  <styles__Button>
     New Item
  </styles__Button>
  <Table
    tableData={Array []}
    tableHeaders={Array []}
  />
</div>
`;
